apps/address-service/domains/common/utils/services/providerDetectors.js

```diff
@@ -34,6 +39,10 @@ function getSearchProvider (args) {
         case GOOGLE_PROVIDER:
             searchProvider = new GoogleSearchProvider(args)
             break
+        case PULLENTI_PROVIDER:
+            logger.warn('⚠️ Pullenti provider still in beta. Normalized result may differ from dadata. Use only for GUID searching.')
```

Comment:
You forgot to pass { msg: '' }, and it is recommended to use lowercase for easier searching
```suggestion
logger.warn({ msg: 'pullenti provider still in beta. Normalized result may differ from dadata, use only for GUID searching.' })
```

Reply:
Oops. Fixed!